.github/issue-labeler-config/regexes-for-issue-labels.yaml

```diff
@@ -0,0 +1,43 @@
+# SPDX-FileCopyrightText: 2009 Fermi Research Alliance, LLC
+# SPDX-License-Identifier: Apache-2.0
+
+# Adds <issue_type> label if issue body contains certain words (when based on the issue template)
+
+# regular expressions for labeling priority of the issue is listed in this block
+Critical:
+  - '\*{2}Priority:\*{2} ?([Cc]ritical|[Uu]rgent)'
+High:
+  - '\*{2}Priority:\*{2} ?[Hh]igh'
+Medium:
+  - '\*{2}Priority:\*{2} ?[Mm]edium'
+Low:
+  - '\*{2}Priority:\*{2} ?[Ll]ow'
+
+# regular expressions for labeling issue type is listed in this block
+BUG:
+  - "bug"
+FEATURE:
+  - "feature( request)?"
+# ENHANCEMENT:
+# IDEA:
+# QUESTION:
+# SUPPORT:
+
+# regular expressions for labeling stakeholders for an issue is listed in this block
+cms:
+  - '\*{2}Stakeholders:\*{2} ?(CMS|Cms|cms)'
+factoryops:
+  - '\*{2}Stakeholders:\*{2} ?([Ff]actory[Oo]ps)'
+fermilab:
+  - '\*{2}Stakeholders:\*{2} ?([Ff]ermilab)'
+fife:
+  - '\*{2}Stakeholders:\*{2} ?(FIFE|Fife|fife)'
+igwn:
+  - '\*{2}Stakeholders:\*{2} ?(IGWN|Igwn|igwn)'
+osg:
+  - '\*{2}Stakeholders:\*{2} ?(OSG|Osg|osg)'
```

Comment:
To get the regular list:
```
cms:
  - '^-   Stakeholder: *(CMS|Cms|cms)'
factoryops:
  - '^-   Stakeholder: *([Ff]actory[Oo]ps)'
...
# Historically IGWN had the LIGO name
igwn:
  - '^-   Stakeholder: *(IGWN|Igwn|igwn|LIGO|Ligo|ligo)'
```
I made changes also to accept 0 or more spaces.
And you could add also the regex for component.

Reply:
Regexes have been updated as suggested and wherever appropriate. Tried to include the `^-   ` part in the regex but upon testing the workflow, noticed that the hyphen at the beginning of line was not detected since that's the markdown syntax. So removed the check for beginning of line in the regexes.